features/dd-sdk-android-ndk/src/test/cpp/integration-tests.cpp

```diff
@@ -1,11 +1,19 @@
+#include <dirent.h>
 #include <jni.h>
 #include <string>
+#include <thread>
+
 
 #include <android/log.h>
 #include <datadog-native-lib.h>
 #include "greatest/greatest.h"
 #include "utils/string-utils.h"
 
+
+#ifndef NDEBUG
```

Comment:
I actually don't think we need this directive check here but just in case

Reply:
We probably do, in fact the whole test would probably need to be `NDEBUG` specific